gradle/libs.versions.toml

```diff
@@ -6,7 +6,7 @@ federation = "3.0.1"
 graphql-java = "20.2"
 graalvm = "0.9.21"
 jackson = "2.15.0"
-kotlin = "1.7.21"
+kotlin = "1.8.21"
```

Comment:
there is a reflection bug in 1.7 that causes runtime failures when trying to access annotation array elements :(